crates/store/re_log_types/src/hash.rs

```diff
@@ -91,7 +86,7 @@ impl Hash128 {
 impl std::hash::Hash for Hash128 {
     #[inline]
     fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
-        state.write_u64(self.0[0]);
+        state.write_u128((self.first64() as u128) << 64 | self.second64() as u128);
```

Comment:
IIRC, you're only allowed to use `write_u64` with `nohash_hasher`, anything else will result in a runtime error in debug builds.

If the hash128 is good, than any 64 bits of it should also be a good hash